cmd/image/qcow2ova/prep/prepare.go

```diff
@@ -21,7 +21,6 @@ import (
 	"strings"
 
 	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
-
```

Comment:
We may need to follow the order of packages :
standard packages.
3rd party. 
internal packages.

Reply:
Modified the order of packages.